File: backend-java/src/main/java/com/citypilot/parking/controllers/HealthController.java
package com.citypilot.parking.controllers;

import com.citypilot.parking.http.HttpResponses;
import com.sun.net.httpserver.HttpExchange;

import java.io.IOException;

import static com.citypilot.parking.utils.JsonUtil.object;

public class HealthController {
    public void health(HttpExchange exchange) throws IOException {
        HttpResponses.json(exchange, 200, object(
                "ok", true,
                "service", "citypilot-parking-api",
                "runtime", "java",
                "version", "0.1.0"
        ));
    }
}

File: backend/controllers/health-controller.js
const { json } = require("../utils/http");

function health(req, res) {
  return json(res, 200, {
    ok: true,
    service: "citypilot-parking-api",
    version: "0.1.0"
  });
}

module.exports = {
  health
};
